### PR TITLE
Updates and content reorg related to the expanded H5 client.

### PR DESCRIPTION
**Pull request by stuclem**
This PR resulted in a bit of general content reorg and consolidation.

- Added info about the contents of the H5 plug-in
- Stated required VC version for H5 plug-in
- Removed more instances where installation of the Flex and H5 clients are mentioned as being separate
- Updated info about how to access VCH Admin
- Reorganized content to separate UI-based VCH admin from VCH admin with `vic-machine`, and to move VCH Admin into a separate section
- Updated the troubleshooting topic on plug-in deployment to mention required VC version

Fixes https://github.com/vmware/vic-product/issues/94

Fixes https://github.com/vmware/vic-product/issues/467

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/access_h5_ui.md [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
 If you have installed the HTML5 plug-in for vSphere Integrated Containers, you can find information about your vSphere Integrated Containers deployment in the HTML5 vSphere Client.
 
 **IMPORTANT**: Do not use the vSphere Client or to perform operations on virtual container host (VCH) appliances or container VMs. Specifically, using the vSphere Client to power off, power on, or delete VCH appliances or container VMs can cause vSphere Integrated Containers Engine to not function correctly. Always use `vic-machine` to perform operations on VCHs. Always use Docker commands to perform operations on containers.
 
+**NOTE**: More functionality will be added to the vSphere Integrated Containers view in future releases.
+
 **Prerequisites**
 
-- You are running vCenter Server 6.5.
+- You are running vCenter Server 6.5.0d or later. The vSphere Integrated Containers view does not function with earlier versions of vCenter Server 6.5.0.
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/access_h5_ui.md]: Technically it does support any version higher than the very first release of 6.5.0 (which means if the user has upgraded to any minor/patch/or whatever version then it should work). But I agree, for the sake of minimizing the risk on our side, that we go this way

File: docs/user_doc/vic_vsphere_admin/ts_ui_not_appearing.md [hunk 2]
```diff
@@ -25,7 +36,7 @@ Restart the client service.
 
 1. Log into the Windows system on which vCenter Server is running.
 2. Open a command prompt as Administrator.
-3. Use the `service-control` command-line utility to stop and then restart the vSphere Client service.<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --stop vspherewebclientsvc</pre><pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --start vspherewebclientsvc</pre>
+3. Use the `service-control` command-line utility to stop and then restart the vSphere Client service.<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --stop vsphere-client</pre><pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --start vsphere-client</pre>
 
 ### Restart the HTML5 Client on a vCenter Server Appliance ###
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/ts_ui_not_appearing.md]: Also I found a typo in "Restart the Flex Client on a vCenter Server Appliance" section where <pre>service vsphere-client stop</pre> probably should have been <pre>service-control --stop vsphere-client</pre>

File: docs/user_doc/vic_vsphere_admin/access_h5_ui.md (hunk 1 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/access_h5_ui.md]: OK, will leave it with 6.5.0d. Thanks!

File: docs/user_doc/vic_vsphere_admin/ts_ui_not_appearing.md (hunk 2 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/ts_ui_not_appearing.md]: Fixed!

**Comment by stuclem** [comment]
Thanks @jooskim !

**Pull request merged**